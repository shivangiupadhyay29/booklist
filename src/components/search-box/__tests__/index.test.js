import React from 'react';
import SearchBox from '../index';
import { shallow } from 'enzyme';

describe('Searchbox component', () => {
  it('unit testing searchbox', () => {
    const mockFnHandler = jest.fn();
    const wrapper = shallow(<SearchBox addSelectedBook={mockFnHandler} />);
    expect(wrapper).toMatchSnapshot();
  });
});
