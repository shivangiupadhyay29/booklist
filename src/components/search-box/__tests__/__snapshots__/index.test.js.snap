// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbox component unit testing searchbox 1`] = `ShallowWrapper {}`;
